### ISSUE TITLE
项目是在那个环境下执行的呀

### ISSUE THREAD

**Issue by bengjagdsajjdglskdajl123424312gfsdfsds** [1xbq1/FedMLLM#1]
可以发一下具体的文件目录的细节吗

**Comment by 1xbq1** [comment]
你好，安装环境参考Install，运行参考Training下的sh finetune_lora.sh，谢谢。

**Comment by 1xbq1** [comment]
用的openbmb/MiniCPM-V-2_6-int4，如不能自动huggingface下载，请手动下载或者采用https://hf-mirror.com，peft：Version: 0.13.2，只在linux系统上运行过

**Comment by bengjagdsajjdglskdajl123424312gfsdfsds** [comment]
flat-attn安装的是哪个版本的呀 发现总是setup不成功 方便说一下吗

**Comment by 1xbq1** [comment]
环境下所有版本如下：
Package                  Version
------------------------ ------------
absl-py                  2.1.0
accelerate               0.30.1
addict                   2.4.0
aiofiles                 23.2.1
annotated-types          0.7.0
anyio                    4.6.0
attrs                    24.2.0
bitsandbytes             0.44.1
bleach                   6.1.0
certifi                  2024.8.30
charset-normalizer       3.3.2
click                    8.1.7
colorama                 0.4.6
contourpy                1.3.0
cycler                   0.12.1
decord                   0.6.0
deepspeed                0.15.1
editdistance             0.6.2
einops                   0.7.0
et-xmlfile               1.1.0
exceptiongroup           1.2.2
fairscale                0.4.0
fastapi                  0.115.0
ffmpy                    0.4.0
filelock                 3.16.1
flash-attn               2.6.3
fonttools                4.54.1
fsspec                   2024.9.0
gradio                   4.41.0
gradio_client            1.3.0
grpcio                   1.66.2
h11                      0.14.0
hjson                    3.1.0
httpcore                 1.0.6
httpx                    0.27.2
huggingface-hub          0.25.1
idna                     3.10
importlib_resources      6.4.5
Jinja2                   3.1.4
joblib                   1.4.2
jsonlines                4.0.0
kaggle                   1.6.17
kiwisolver               1.4.7
lxml                     5.3.0
Markdown                 3.7
markdown-it-py           3.0.0
markdown2                2.4.10
MarkupSafe               2.1.5
matplotlib               3.7.4
mdurl                    0.1.2
modelscope_studio        0.4.0.9
more-itertools           10.1.0
mpmath                   1.3.0
networkx                 3.3
ninja                    1.11.1.1
nltk                     3.8.1
numpy                    1.24.4
nvidia-cublas-cu12       12.1.3.1
nvidia-cuda-cupti-cu12   12.1.105
nvidia-cuda-nvrtc-cu12   12.1.105
nvidia-cuda-runtime-cu12 12.1.105
nvidia-cudnn-cu12        8.9.2.26
nvidia-cufft-cu12        11.0.2.54
nvidia-curand-cu12       10.3.2.106
nvidia-cusolver-cu12     11.4.5.107
nvidia-cusparse-cu12     12.1.0.106
nvidia-ml-py             12.560.30
nvidia-nccl-cu12         2.18.1
nvidia-nvjitlink-cu12    12.6.77
nvidia-nvtx-cu12         12.1.105
opencv-python-headless   4.5.5.64
openpyxl                 3.1.2
orjson                   3.10.7
packaging                23.2
pandas                   2.2.3
peft                     0.13.0
Pillow                   10.1.0
pip                      24.2
portalocker              2.10.1
protobuf                 4.25.0
psutil                   6.0.0
py-cpuinfo               9.0.0
pydantic                 2.9.2
pydantic_core            2.23.4
pydub                    0.25.1
Pygments                 2.18.0
pyparsing                3.1.4
python-dateutil          2.9.0.post0
python-multipart         0.0.12
python-slugify           8.0.4
pytz                     2024.2
PyYAML                   6.0.2
regex                    2024.9.11
requests                 2.32.3
rich                     13.9.2
ruff                     0.6.9
sacrebleu                2.3.2
safetensors              0.4.5
scikit-learn             1.5.2
scipy                    1.14.1
seaborn                  0.13.0
semantic-version         2.10.0
sentencepiece            0.1.99
setuptools               75.1.0
shellingham              1.5.4
shortuuid                1.0.11
six                      1.16.0
sniffio                  1.3.1
socksio                  1.0.0
starlette                0.38.6
sympy                    1.13.3
tabulate                 0.9.0
tensorboard              2.18.0
tensorboard-data-server  0.7.2
tensorboardX             2.6.2.2
text-unidecode           1.3
threadpoolctl            3.5.0
timm                     0.9.10
tokenizers               0.19.1
tomlkit                  0.12.0
torch                    2.1.2
torchvision              0.16.2
tqdm                     4.66.1
transformers             4.40.0
triton                   2.1.0
typer                    0.12.5
typing_extensions        4.8.0
tzdata                   2024.2
urllib3                  2.2.3
uvicorn                  0.24.0.post1
webencodings             0.5.1
websockets               12.0
Werkzeug                 3.0.4
wheel                    0.44.0
